src/iohumdrum.cpp

```diff
@@ -671,22 +756,22 @@ void HumdrumInput::getRespPeople(vector<vector<string> > &respPeople, vector<Hum
 {
 
     // precalculate a reference map here to make more O(N) rather than O(N^2)
-    addPerson(respPeople, references, "COM", "Composer"); // cmp
+    addPerson(respPeople, references, "COM", "composer"); // cmp
```

Comment:
Lower case is fine, but remember that I was using upper case to indicate these are authority labels, and the lower-case ones were labels that I defined since I couldn't find a match in the LOC list.